package.json

```diff
@@ -86,7 +86,7 @@
 		"env:stop": "npm run wp-env stop",
 		"test:help": "wp-scripts test-unit-js --help",
 		"test:performance": "npm run wp-env:config && cross-env NODE_CONFIG_DIR=tests/e2e/config wp-scripts test-e2e --config tests/e2e/config/jest.performance.config.js -- performance",
-		"test:php": "npm run wp-env run phpunit '/var/www/html/wp-content/plugins/${PWD##*/}/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/${PWD##*/}/phpunit.xml.dist'",
+		"test:php": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/${PWD##*/}/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/${PWD##*/}/phpunit.xml.dist",
```

Comment:
Just a small thing, but, [you can use `--env-cwd` now](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/#wp-env-run-container-command) to clean up this command!